### ISSUE TITLE
Training stages

### ISSUE THREAD

**Issue by davidelobba** [BoyuanJiang/FitDiT#26]
Hi, thank you very much for your amazing work! I have few questions regarding the training stages.

When you finetune the GarmentDiT and the DenoisingDiT, do you finetune the whole model or only specific parts of it?
Regarding the second stage, what is the dropout probability of pooled_projections and garment latents? And have you trained the pose guider during the second stage together with the DenoisingDiT?

Thank you again!
